tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrAllowed.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %dxc -T lib_6_7 %s | FileCheck %s
```

Comment:
/validation/ isn't the right place for any of these. I suspect they should all be verification tests even the ones that just verify that no error is produced